src/main/java/in/projecteka/consentmanager/user/UserService.java

```diff
@@ -118,4 +166,8 @@ private boolean validateOtpVerification(OtpVerification otpVerification) {
     public String getUserIdSuffix() {
         return userServiceProperties.getUserIdSuffix();
     }
+
+    public int getExpiryInMinutes() {
+        return otpServiceProperties.getExpiryInMinutes();
+    }
```

Comment:
Is is needed to be public?

Reply:
Yes. otpServiceProperties are part of userService. We are only exposing the property getter here, not the properties.

Reply:
But if we using `getExpiryInMinutes` in some other class then why not directly use `otpServiceProperties.getExpiryInMinutes();`  in that class. And if we are using this method only in this class then it can be private.

Reply:
otpServiceProperties are related to the user service alone. OtpServiceProperties are injected to the specific services upon start of the application.